### ISSUE TITLE
~~pressing "j" or "k" in insert mode brings up whichkey, so I can never enter a j or k character~~ I hate the "I hate escape" part of the config

### ISSUE THREAD

**Issue by Araly** [ChristianChiarulli/LunarVim#377]
I was writing an issue, then I was able to fix it, but I'll still write the issue to help people that might stumble on the same problem I had.

Whenever I was in insert mode, pressing j or k would make whichkey pop up, which made me think I couldn't write any j or k. The reality is that I can just write normally and ignore the whichkey prompt, but it's very confusing when you don't know what's going on.

I disabled the "I hate escape" part of the config, because I like escape, and I hate whichkey prompts when I'm trying to write something.

That's all really, not really an issue, more a "this part of the config was confusing and made me go back to Codium for a bit, thinking that lunarvim wasn't ready for use yet". I've been loving lunarvim apart from that, it feels like a simpler and easier to understand spacemacs to me.

**Comment by rebuilt** [comment]
You can increase timeoutlen in the settings file to your liking.  This will delay the amount of time whichkey waits before popping up.  If you don't want it showing up so quickly, increase this value.  It's measured in milliseconds.  1000ms = 1 second.  Chris added a setting for it in lv-settings yesterday but it doesn't work correctly because that file gets loaded after the setting has already been made.  So change it directly in the settings.lua file.

**Comment by folke** [comment]
@Araly I actually made a change two days ago to explicitly never hook on `j` or `k` in `INSERT` mode, since as you're saying this might be very confusing for people.

Are you sure you're using the latest version of **WhichKey**?

**Comment by jordisantamaria** [comment]
> @Araly I actually made a [change](https://github.com/folke/which-key.nvim/commit/9a2faed055459d3226634344468f78bf85d77fa8) 6 days ago to explicitly never hook on `j` or `k` in `INSERT` mode, since as you're saying this might be very confusing for people.
> 
> Are you sure you're using the latest version of **WhichKey**?

The change stops showing the popup of whichkey, but still If you write kjk, k isn't written. So you are forced to do k "wait" then write jk, If you want k to be write.

**Comment by folke** [comment]
@jordisantamaria that has indeed nothing todo with WhichKey. This is how vim works. When you type `kjk`, on the last `k`, vim waits to see if you meant just `k` or `kj`.

So even when fully disabling WK, the behavior would be the same.

**Comment by jordisantamaria** [comment]
Yes sorry I see, I don't know why I though  before It was not happening, but seems It have nothing to do which whichkey plugin.
